### ISSUE TITLE
Improve understanding of link purpose

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#266]
## Is your feature request related to a problem? Please describe.
- The Schedule a Visit button doesn't indicate that it will open an email, which should be better clarified particularly for screenreader users.
- In footer, "By appointment only" link text is not descriptive of its target

## Describe the solution you'd like
- Change the “Schedule a visit” button to be a link with `title=“opens email”`. Alternatively, if we want to keep this as a button, we can add an aria-label that gives a better indication of what will happen if the user presses it - "Email to schedule a visit."
- In Footer, either change "By appointment only" link text or add an aria-label to the link that reads something like "How to access and request materials."

## Additional context
See [WCAG SC 2.4.4. - Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
